matrix/common/dynamo_handler.py

```diff
@@ -44,6 +44,20 @@ def __init__(self):
         self._state_table = self._dynamo.Table(DynamoTable.STATE_TABLE.value)
         self._output_table = self._dynamo.Table(DynamoTable.OUTPUT_TABLE.value)
 
+    def _get_dynamo_table_from_name(self, dynamo_table_name: str):
```

Comment:
Personally think name is verbose for this impl, but would be appropriate if `_get_dynamo_table...` was overloaded. Would opt for `_get_dynamo_table`

Reply:
`dynamo_table_name` should use enum and function impl accesses enum `.value`

Reply:
Oh I see what you're doing now. I'm fine with this, but I think using the enum object the whole time might be cleaner